Changelog.md

```diff
@@ -1,3 +1,6 @@
+**V1.9.17 - Updates**
+- Fix southern hemisphere -12hr RA offset
```

Comment:
Could you change to clarify that this is just a reporting issue and not a slewing one?

Reply:
Hopefully it's clearer now